apps/meteor/app/slackbridge/server/SlackAPI.js

```diff
@@ -40,102 +40,133 @@ export class SlackAPI {
+				members = members.concat(response.members);
+				const hasMoreItems = response.response_metadata && response.response_metadata.next_cursor;
 				if (hasMoreItems) {
-					currentCursor = response.data.response_metadata.next_cursor;
+					currentCursor = response.response_metadata.next_cursor;
 				}
 			}
 		}
 		return members;
 	}
 
-	react(data) {
-		const response = HTTP.post('https://slack.com/api/reactions.add', { params: data });
-		return response && response.statusCode === 200 && response.data && response.data.ok;
+	async react(data) {
+		const request = await fetch('https://slack.com/api/reactions.add', {
+			method: 'POST',
+			body: data,
+		});
+		const response = await request.json();
+		return response && request.status === 200 && response && request.ok;
 	}
 
-	removeReaction(data) {
-		const response = HTTP.post('https://slack.com/api/reactions.remove', { params: data });
-		return response && response.statusCode === 200 && response.data && response.data.ok;
+	async removeReaction(data) {
+		const request = await fetch('https://slack.com/api/reactions.remove', {
+			method: 'POST',
+			body: data,
+		});
+		const response = await request.json();
+		return response && request.status === 200 && response && request.ok;
 	}
 
-	removeMessage(data) {
-		const response = HTTP.post('https://slack.com/api/chat.delete', { params: data });
-		return response && response.statusCode === 200 && response.data && response.data.ok;
+	async removeMessage(data) {
+		const request = await fetch('https://slack.com/api/chat.delete', {
+			method: 'POST',
+			body: data,
+		});
+		const response = await request.json();
+		return response && request.status === 200 && response && request.ok;
 	}
 
-	sendMessage(data) {
-		return HTTP.post('https://slack.com/api/chat.postMessage', { params: data });
+	async sendMessage(data) {
+		const request = await fetch('https://slack.com/api/chat.postMessage', {
+			method: 'POST',
+			body: data,
+		});
+		return request.json();
 	}
 
-	updateMessage(data) {
-		const response = HTTP.post('https://slack.com/api/chat.update', { params: data });
-		return response && response.statusCode === 200 && response.data && response.data.ok;
+	async updateMessage(data) {
+		const request = await fetch('https://slack.com/api/chat.update', {
+			method: 'POST',
+			body: data,
+		});
+		const response = await request.json();
+		return response && request.status === 200 && response && request.ok;
 	}
 
-	getHistory(family, options) {
-		const response = HTTP.get(`https://slack.com/api/${family}.history`, {
-			params: Object.assign({ token: this.apiToken }, options),
+	async getHistory(family, options) {
+		const request = await fetch(`https://slack.com/api/${family}.history`, {
+			params: {
+				token: this.apiToken,
+				...options,
+			},
 		});
-		return response && response.data;
+		const response = await request.json();
```

Comment:
```suggestion
		return request.json();
```

Reply:
There's no actual perf reasons to not use backticks, but I'll change them :)